### ISSUE TITLE
Fan switch does not change state when furnace fan is on.

### ISSUE THREAD

**Issue by jbrovage** [chrisjshull/homebridge-nest#464]
**Describe the bug**
When building an automation dependent on the "fan" switch's status, the automation will not work, as the "fan" switch always reports as off, if the fan was turned on by the thermostat itself.  Activating the furnace fan switch manually does change the status.

**To Reproduce**
1.  monitor fan switch status
2. adjust thermostat temperature to force furnace to turn on.
2. check fan switch status
3. fan switch reports as "off"

**Expected behavior**
fan switch state should go from "off" to "on" whenever the thermostat tells the furnace/AC to activate.  

Include with your bug report this version info:
```sh
node  --version
v16.13
homebridge --version
1.3.6```

Make sure you have the latest LTS from https://nodejs.org
and the latest packages: `npm upgrade -g homebridge homebridge-nest`

Also include debug log output from startup through seeing the issue:
`DEBUG=* homebridge -D`

**Comment by adriancable** [comment]
@jbrovage - the fan switch only shows 'on' when the fan is started by the user, either manually or via the schedule set in the Nest app.

In general, the thermostat doesn't know when the fan is running, unless the thermostat itself is controlling the fan. For example, a lot of HVAC systems will automatically run the fan when the system is cooling. This is handled on the HVAC system's control board, and the thermostat is completely unaware of it.

So the solution is to simply note that automations triggered by the fan control will only run when you (rather than other parts of the HVAC system) manually turn on the fan. If you also want to run an automation when the fan is turned on due to some other cause, you'll need to set up a specific automation for that using the correct trigger. For example, if your HVAC system always runs the fan when it's cooling, then the right automation trigger for that is 'Current Heating/Cooling State' is 'Cooling'. You can set an automation like this using a third-party app (like the excellent Home+) which allows automations to be created on any characteristic (vs. the Apple Home app which supports only a limited subset).

So in summary: this is working as designed. Hope this helps.
